src/hawkmoth/__init__.py

```diff
@@ -155,13 +155,13 @@ def __get_docstrings(self, viewlist):
 
         num_matches = 0
         for root in self.__parsed_files():
-            if filter_filenames is not None and root.get_filename() not in filter_filenames:
+            if self._skip(root.get_filename(), filter_filenames):
```

Comment:
Hmm, I find the imbalance a bit triggering. The other ones gained specific `skip_something` methods, these ones use the `_skip` one.

I get why you did it this way here, but it's still a bit of cognitive load spent understanding the difference. I'd be tempted at least to remove the other `skip_*` ones and just call `_skip` everywhere.